viadot/sources/duckdb.py

```diff
@@ -172,6 +172,9 @@ def create_table_from_parquet(
                 return True
             elif if_exists == "delete":
                 self.run(f"DELETE FROM {fqn}")
+                self.logger.info(f"Successfully deleted data from table '{fqn}'.")
+                self.run(f"INSERT INTO {fqn} SELECT * FROM read_parquet('{path}')")
+                self.logger.info(f"Successfully inserted data into table '{fqn}'.")
```

Comment:
Does this make sense here considering we run below just after these lines? Don't these lines override your code? It seems if we always run a CTAS, then the "delete" option doesn't make sense at all since we're always creating a new table from scratch.
https://github.com/dyvenia/viadot/blob/d73d6e5ff698b4dfae54c7cde2fb1d65635cf3d7/viadot/sources/duckdb.py#L189-L190

Reply:
In this case we are not creating table because we finish code execution with return in line 178